inference-engine/cmake/features_ie.cmake

```diff
@@ -74,13 +74,13 @@ ie_dependent_option (GAPI_TEST_PERF "if GAPI unit tests should examine performan
 
 ie_dependent_option (ENABLE_MYRIAD_MVNC_TESTS "functional and behavior tests for mvnc api" OFF "ENABLE_TESTS;ENABLE_MYRIAD" OFF)
 
-ie_dependent_option (ENABLE_DATA "fetch models from testdata repo" ON "ENABLE_FUNCTIONAL_TESTS;NOT ANDROID" OFF)
+ie_dependent_option (ENABLE_DATA "fetch models from testdata repo" ON "ENABLE_FUNCTIONAL_TESTS;NOT ANDROID" ON)
```

Comment:
Does it mean that the data is downloaded every cmake run by default now?
In my opinion the date should not be downloaded by default

Reply:
it will be downloaded only once and updated once repo has new commits.